src/views/product-root-docs-path-landing/server.ts

```diff
@@ -145,6 +145,22 @@ const getStaticProps = async (context: GetStaticPropsContext) => {
 		return getStaticPropsResult
 	}
 
+	/**
+	 * TODO: Remove this when (HCP) Waypoint IA is updated
+	 */
+	if (product.slug === 'waypoint') {
+		getStaticPropsResult.props.layoutProps['sidebarNavDataLevels'] =
+			getStaticPropsResult.props.layoutProps.sidebarNavDataLevels.map(
+				(navLevel) => {
+					delete navLevel.menuItems
```

Comment:
used `delete` here rather than `menuItems: []` below because it was causing a TypeError and I didn't want to make more changes such as moving types from Sidebar component repo or adding a new one.

Reply:
This seems like a fine choice, especially because this is a temp change.